packages/fxa-payments-server/server/lib/amplitude.test.js

```diff
@@ -210,6 +210,7 @@ describe('lib/amplitude', () => {
         device_id: 'QUUX',
         session_id: 1570000000000,
         app_version: '148.8',
+        country_iso: "DE",
```

Comment:
I'm going to make a unilateral decision here and ask you to rename `country_iso` to `country_code`. I think it's much more clear, and we should do it now when we're first adding the field. I will update the ticket and the tech spec.